src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -513,17 +513,16 @@ private function isMaxDepthReached(array $attributesMetadata, string $class, str
      *
      * {@inheritdoc}
      */
-    protected function createChildContext(array $parentContext, $attribute/*, string $format = null */)
+    protected function createChildContext(array $parentContext, $attribute/*, ?string $format */)
```

Comment:
the new argument should be documented: `@param string|null $format` (same in AbstractNormalizer, to make it clear to DebugClassLoader that the missing argument on the signature is known and shouldn't trigger a deprecation notice)

Reply:
fixed